test/gateways/authorize_net_test.exs

```diff
@@ -64,4 +80,120 @@ defmodule Gringotts.Gateways.AuthorizeNetTest do
     end
   end
 
+  describe "authorize" do
+    test "successful response" do
```

Comment:
same here should convey something about the scenario of what we are passing and then result.